src/pages/api/token-details.ts

```diff
@@ -0,0 +1,32 @@
+import config from '@/config'
+import type { NextApiRequest, NextApiResponse } from 'next'
+
+type ResponseData = {
+  status: boolean
+  message: string
+  response?: any
+}
+
+export default async function handler(
+  req: NextApiRequest,
+  res: NextApiResponse<ResponseData>,
+) {
+  const { tokenAddress, chain } = req.query
+  if (!tokenAddress || !chain) {
+    return res
+      .status(400)
+      .json({ status: false, message: 'Token address and chain id are needed' })
+  }
+  const url = `https://pro-openapi.debank.com/v1/user/token_list?id=${tokenAddress}&chain_id=${chain}&is_all=true`
+  const result = await fetch(url, {
+    headers: {
+      Accept: 'application/json',
+      Accesskey: `${config.debankApiKey}`,
+    },
+  })
+  return res.status(200).json({
```

Comment:
this api has not any cache, please review 
https://vercel.com/docs/concepts/functions/serverless-functions/edge-caching
https://vercel.com/docs/concepts/edge-network/caching

Reply:
@kesar, we can cache for a day right?

Reply:
yeah 👍🏻